### ISSUE TITLE
json-editor fails on load when server side rendering is used

### ISSUE THREAD

**Issue by Krivich** [json-editor/json-editor#1163]
### General information

- json-editor version: 2.5.3

### Expected behavior

Browser and Server Side Rendering modes are working identically 

### Actual behavior

ReferenceError: window is not defined

### Steps to reproduce the behavior

1. Import json-editor into test component of Angular application with 
`import {JSONEditor} from '@json-editor/json-editor/dist/nonmin/jsoneditor.js';`
2. Start Angular locally in ssr mode
3. Open test page in browser
4. Find error message in the log `ReferenceError: window is not defined`

The problem is caused by referencing of `window` by webpack in the webpackUniversalModuleDefinition. The similar webpack problem is discussed here https://github.com/webpack/webpack/issues/6784

**Comment by Krivich** [comment]
Hi. The problem is than the existing webpack configs are designed for browser only. To solve this problem I tried to create and register webpack.nodejs.js with custom options for nodejs environment. 
The first problem which stops loading of json-editor was solved by  this suggestion: https://github.com/webpack/webpack/issues/6784#issuecomment-375941431
After that loading failed on the next step because in the nodejs env `file-loader` should be used instead of  `style-loader`:
https://github.com/webpack-contrib/style-loader/issues/461#issuecomment-593957042. I tried to apply this suggestion here https://github.com/Krivich/json-editor/blob/json-editor_in_nodejs_env/config/webpack.nodejs.js but didn't make it compliable.

In my case I decided to implement lazy-loading wrapper around my json-editor Angular component to avoid loading of json-editor library in nodejs env - it works fine but ssr-rendered page has no json-editor component, acceptable for my case.
